.travis.yml

```diff
@@ -23,6 +23,8 @@ before_script:
 script:
     # flake8 the diff
     - "git diff HEAD^ '*.py' | flake8 --diff --config flake8.cfg"
+    # add secret key to django settings.py
+    - echo "SECRET_KEY= '$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 100 | head -n 1)'" >> platform/pulpcore/app/settings.py
```

Comment:
From what I can see, Django secret keys also contain special characters.  Would the lack thereof here compromise the integrity of the key?

I believe the full set of possible characters is 

abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)